install_config/install/prerequisites.adoc

```diff
@@ -806,6 +806,53 @@ See
 link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Logical_Volume_Manager_Administration/index.html[Logical
 Volume Manager Administration] for more detailed information on LVM management.
 
+[[managing-docker-container-logs]]
+*Managing Docker Container Logs*
+
+Sometimes a container's log file
+(*_/var/lib/docker/containers/HASH/HASH-json.log_*) can increase to a
+problematic size. You can manage this by configuring the `json-file` logging
+driver to restrict the size and number of log files.
+
+For example, `--log-opt max-size` sets the size at which a new log file is
+created. `--log-opt max-file` is the maximum number of files that a log is
+rolled over before being discarded. You can use these options with Docker 1.8.2
+or later.
+
+Edit the *_/etc/sysconfig/docker_* file to configure `max-size=1M` and
```

Comment:
"file on each host to configure these options. For example:"

I feel like currently it makes it seem like we're suggesting these particular values. Could also lead in to the example with:

"For example, to set the maximum file size to 1MB and always keep the last three log files:"